atlas/main.py

```diff
@@ -58,11 +66,17 @@
             for link in links
             if link.text.lower() == "documents"
         ]
+
         data.append(
             dict(
                 title=short_title.text,
+                physics_group=physics_group.text,
+                date=date.text,
+                com_energy_tev=com_energy_tev.text,
+                luminosity=luminosity.text,
```

Comment:
As these are elements in a JSON format I suppose that it would be neater to have these as `-` separated instead of `_`.

```suggestion
                physics-group=physics_group.text,
                date=date.text,
                com-energy-tev=com_energy_tev.text,
                luminosity=luminosity.text,
```